cluster-scope/overlays/prod/moc/infra/kustomization.yaml

```diff
@@ -4,9 +4,6 @@ kind: Kustomization
 
 resources:
   - ../common
-
```

Comment:
This appears to remove all the argocd resources from the infra cluster, without adding them back anywhere else?

Reply:
Or these aren't necessary here, because we add them from `argocd/cluster-resources/kustomization.yaml`, and the entries in the infra overlay were redundant?

Reply:
Yep it's the latter -- I added them to the argocd overlay kustomization. It made sense to have them there in my mind, because we need to be able to install argocd fully by using a manual `oc apply`.